README.md

```diff
@@ -43,120 +36,74 @@ Check the original technical report
 | summary                 | string  | Summary of the layers, can be the output of `print(model)`. |
 | pretrained              | string  | Indicates the source of the pretraining (ex: ImageNet).     |
-
-### Runtime Object
-
-| Field Name        | Type                               | Description                                                                  |
-|-------------------|------------------------------------|------------------------------------------------------------------------------|
-| framework         | string                             | Used framework (ex: PyTorch, TensorFlow).                                    |
-| version           | string                             | Framework version (some models require a specific version of the framework). |
-| model_handler     | string                             | Inference execution function.                                                |
-| model_src_url     | string                             | Url of the source code (ex: GitHub repo).                                    |
-| model_commit_hash | string                             | Hash value pointing to a specific version of the code.                       |
-| docker            | \[[Docker Object](#docker-object)] | Information for the deployment of the model in a docker instance.            |
-
-#### Docker Object
+| total_parameters        | integer | Total number of parameters.                                 |
+| on_disk_size_mb         | number  | The memory size on disk of the model artifact (MB).         |
+| ram_size_mb | number    | number  | The memory size in accelerator memory during inference (MB).|
+
+### Runtime
+
+| Field Name            | Type                               | Description                                                                              |
+|-----------------------|------------------------------------|------------------------------------------------------------------------------------------|
+| framework             | string                             | Used framework (ex: PyTorch, TensorFlow).                                                |
+| version               | string                             | Framework version (some models require a specific version of the framework).             |
+| model_asset        | [Asset Object](https://github.com/radiantearth/stac-spec/blob/master/collection-spec/collection-spec.md#asset-object)                        | Common Metadata Collection level asset object containing URI to the model file.   |
+| model_handler         | string                             | Inference execution function.                                                            |
+| source_code_url         | string                             | Url of the source code (ex: GitHub repo).                                                |
```

Comment:
are you saying the  model handler should be an Asset? or that the whole runtime object could be an Asset object?

Reply:
The `source_code_url` could provide more details than simply the URL. For instance, it could provide the media-type as HTML if it is a GitHub repo link, or ZIP if it points to an archived revision. The title, description and role properties could also be relevant to better describe it.

Reply:
done